services/users/src/config.js

```diff
@@ -1,6 +1,22 @@
-'use strict';
+export const PREFIX = '/api/v1';
+export const SALT = 7;
 
-const PREFIX = '/api/v1';
-const SALT = 7;
+export const kafkaTopics = {
+  SUCCESSFUL_REGISTRATION: 'new-user-registered',
```

Comment:
I suggest renaming it to clarify the topic name
```suggestion
  NEW_USER_REGISTERED: 'new-user-registered',
```